src/app/core/parsers/zod-rundown-event-parser.service.ts

```diff
@@ -0,0 +1,97 @@
+import { Injectable } from '@angular/core'
+import {
+    RundownInfinitePieceAddedEvent,
+    RundownActivatedEvent,
+    RundownDeactivatedEvent,
+    RundownResetEvent,
+    PartSetAsNextEvent,
+    PartTakenEvent, RundownAdLibPieceInsertedEvent, RundownDeletedEvent
+} from '../models/rundown-event'
+import * as zod from 'zod'
+import { RundownEventType } from '../models/rundown-event-type'
+import { EntityParserService } from '../abstractions/entity-parser.service'
+import { AdLibPiece } from '../models/ad-lib-piece'
+
+@Injectable()
+export class ZodRundownEventParser {
```

Comment:
Same comments in this file as for `zod-entiry-parser.service`

Reply:
Same solutions as for `zod-entity-parser.service` are applied.